reframe/core/containers.py

```diff
@@ -29,25 +28,27 @@ class ContainerPlatform(abc.ABC):
     #: If no command is given, then the default command of the corresponding
     #: container image is going to be executed.
     #:
-    #: ..versionchanged:: 3.5
+    #: ..versionadded:: 3.5.0
```

Comment:
You should leave a space after `..`. Otherwise it is not shown correctly. Make sure to fix this everywhere.